packages/kit/src/views/Swap/components/SwapPercentageStageBadge.tsx

```
@@ -1,41 +1,56 @@
-    {...props}
-  >
-    <Badge.Text size="$bodySmMedium" color="$textSubdued">
-      {stage}%
-    </Badge.Text>
-  </Badge>
-);
+} & IStackProps) => {
+  const component = (
+    <Badge
+      key={key}
+      role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)